learn/experimental/autobatching.md

```diff
@@ -0,0 +1,34 @@
+# Auto-batching
+
+If enabled, this feature automatically combines multiple document addition requests ([`POST`](/reference/api/documents.md#add-or-replace-documents) or [`PUT`](/reference/api/documents.md#add-or-update-documents)) into batches to speed up indexation.
+
```

Comment:
Should we mention the requirements for documentation additions to be part of the same batch?

From the [github discussion](https://github.com/meilisearch/meilisearch/discussions/2070):

>Auto-batching is a feature that will batch together consecutive document additions and perform them together to improve indexing speed. For document addition to being added to the same batch, they need to:
>- target the same index
>- have the same update method (POST /documents or PUT /documents, i.e update or replace documents)
>- be immediately consecutive